uint64 id
uint32 object_class
geometry_msgs/PointStamped space_loc
